lib/middleware/fetch-open-tasks.js

```diff
@@ -1,10 +1,14 @@
-const { get } = require('lodash');
+const { get, isFunction } = require('lodash');
 
 module.exports = id => (req, res, next) => {
   if (!id && !get(res, 'response.id')) {
     return next();
   }
 
+  if (isFunction(id)) {
+    id = id(req);
```

Comment:
```suggestion
    id = id(req. res);
```

Even if we're not using it downstream right now it's generally good form to always pass both so things work predictably.